back-end/src/helpers/connect-to-mongo.ts

```diff
@@ -2,7 +2,7 @@ import mongoose from 'mongoose';
 import config from 'config';
 
 export async function connectToMongo() {
-  const baseUrl = process.env.CI_RUNNER ? 'mongodb://mongo' : 'mongodb://127.0.0.1';
+  const baseUrl = (process.env.CI_RUNNER ? 'mongodb://mongo' : (config.has('DB_HOST') ? `mongodb://${config.get('DB_HOST')}` : 'mongodb://127.0.0.1'));
```

Comment:
No need to check if `config.has('DB_HOST')` since it's set in the default.yml file - it should always be set.

If it is not set, it will throw an error, which is fine since we don't expect that to occur.

Reply:
Fair, I was just being safe.

Reply:
it's mainly to keep the code readable, since multiple ternary paths is hard to read, best to avoid if possible :)